com.unity.cinemachine/Editor/Upgrader/CinemachineUpgrader.cs

```diff
@@ -196,9 +196,14 @@ public static bool Upgrade(GameObject old)
                     if (vcam != null)
                         result = UpgradeVcam(vcam);
                 }
+
+                // Cinemachine pipeline components (there will be more of these...)
                 old.TryGetComponent<CinemachineComposer>(out var composer);
                 if (composer != null)
                     UpgradeComposer(composer);
+                old.TryGetComponent<CinemachineFramingTransposer>(out var ft);
+                if (ft != null)
+                    UpgradeFramingTransposer(ft);
```

Comment:
This could also be in UpgradeComponent like TrackedDolly -> SplineDolly or that should live out here.